README.md

```diff
@@ -154,7 +155,54 @@ export BB_ENV_PASSTHROUGH_ADDITIONS="PROJECT MACHINE"
 2. Execute `source yocto/source/oe-init-build-env build` which initializes the build environment and changes the directory to `yocto/build`.
 3. Execute `bitbake core-image-minimal` to build the image.
 
-The resulting image will be found in `yocto/build/tmp/deploy/image/<machine>`.
+The resulting image will be found in `yocto/build/tmp/deploy/image/<machine>` have a filename similar to `core-image-minimal-platform-1234567890-rootfs.ext4.gz`
+
+### Flashing an image <a name="flash"></a>
+
+The internal storage of the products Charge Control C and M is divided into several partitions and uses rauc to handle updates. If you are interested in adding rauc to your own image you can take a look at raucs documentation https://rauc.readthedocs.io/en/latest/integration.html#yocto .To flash an image it is important to identify the currently not in use partition because this will be our target partition. The following tables describe the partitioning for each Charge Control M and Charge Control C:
+
+**Charge Control M:**
+| **`Partition`** | **`Size`**  | Description |
```

Comment:
formatting nitpicks: partition and size is both, marked as bold and code; description neither nor, I'd suggest to drop both